### ISSUE TITLE
HOW TO TUNE A RECOMMENDATION ENGINE TO FORECAST FUTURE TRENDS/OCCURRENCES

### ISSUE THREAD

**Issue by aidelojep** [section-engineering-education/engineering-education#2715]
###HOW TO TUNE A RECOMMENDATION ENGINE TO FORECAST FUTURE TRENDS/OCCURRENCES

### RECOMMENDER SYSTEM is a system widely used by online retailers and various business platforms to make suggestions to a pool of customer base in making a more likely selection from a list of items based on previous history or digital footprints.
This model is very helpful for developers who might choose to include this system in the overall infrastructural design for recent websites and e-commerce platforms.

### Key takeaways: 
(1.) Understanding of the basic concept of RECOMMENDATION ENGINE and how it works.
(2.) Step-by-step tutorial guide on how to build a RECOMMENDATION ENGINE.
(3.) Algorithms used to build the RECOMMENDATION ENGINE like Classification model.
(4.) Fine-tuning the recommendation model.

### Templates to use as guides
-Introduction
-Table of contents
-Prerequisite
-Goal of tutorial
-Steps to follow 
-Conclusion

**Comment by lalith1403** [comment]
I have a few questions. 

1. What model are you using to train the model?
2. What dataset are you using?
3. Are you building a full stack app? If yes, what are you using for Frontend and backend. 
4. I would stick to the project rather than cover introduction to ml, and the introduction of various other models. 

The approach we take per article is to go in depth versus cover a lot of points. 

@hectorkambow

**Comment by aidelojep** [comment]
Dear Section-Engineering-Education/Engineering-Education,

Thank you for your feedback. In response to your email, I have responded to
all your concerns below:

1.) This proposed project will be trained on the Tensorflow model.
2.)  The dataset is an inbuilt digit dataset gotten from Karas.
3.)  Yes, this project will run on a backend (using FastApi ) and CSS for
the FrontEnd.
      I understand exactly what you mean by going in-depth rather than
touching on the surface of several other side topics. The project will be
implemented with a key focus on the subject area.
I await your approval/confirmation to proceed.

Best regards.

~Peter Lucky Aideloje.

**Comment by aidelojep** [comment]
Dear Lalith,

Thank you for your feedback. In response to your questions, I have responded to all your concerns below:

1.) This proposed project will be trained on the Tensorflow model.
2.)  The dataset is an inbuilt digit dataset gotten from Karas.
3.)  Yes, this project will run on a backend (using FastApi ) and CSS for the FrontEnd.
      I understand exactly what you mean by going in-depth rather than touching on the surface of several other side topics. The project will be implemented with a key focus on the subject area.
I await your approval/confirmation to proceed.

**Comment by lalith1403** [comment]
Hello @aidelojep. I feel this is a very basic and widely covered topic that wouldn't add value to our readers. I suggest you to come up with a more unique article. Do let me know if you have any doubts.

**Comment by hectorkambow** [comment]
@lalith1403 

Be sure to leave a questions label - and follow up if needed 👍

**Comment by aidelojep** [comment]
@louisefindlay23 Thank you for your feedback, I intend to write a bit further on these topics option: 
1.) Using Digit prediction model to facial recognition.
2.) Tunning a recommendation Engine to forecast future trends/occurrences.
3.) Building a Bulk SMS API with Django and Twilio API.
I am well versed in the above topics. Kindly let me know what you think.
Cheers.

**Comment by aidelojep** [comment]
@lalith1403 What is the update on this/

**Comment by lalith1403** [comment]
I am sorry I missed this issue. Well, can you elaborate on what project out of the 3 points mentioned will you be working on? This seems to be a diverse set of projects. 

You may look at approved issues to get more clarity on the type of topic proposals we expect.

**Comment by aidelojep** [comment]
@lalith1403 thank you for your continued assistance. Please find below a recent detailed topic on recommendation model I want to elaborate on. I await your positive feedbacks. 

HOW TO TUNE A RECOMMENDATION ENGINE TO FORECAST FUTURE TRENDS/OCCURRENCES
RECOMMENDER SYSTEM is a system widely used by online retailers and various business platforms to make suggestions to a pool of customer base in making a more likely selection from a list of items based on previous history or digital footprints. 
This model is very helpful for developers who might choose to include this system in the overall infrastructural design for recent websites and e-commerce platforms.
Key takeaways:
(1.) Understanding of the basic concept of RECOMMENDATION ENGINE and how it works.
(2.) Step-by-step tutorial guide on how to build a RECOMMENDATION ENGINE.
(3.) Algorithms used to build the RECOMMENDATION ENGINE like Classification model.
(4.) Fine-tuning the recommendation model. 

Templates to use as guides
-Introduction
-Table of contents
-Prerequisite
-Goal of tutorial
-Steps to follow
-Conclusion

**Comment by lalith1403** [comment]
Update the contents of the initial issue with the updated contents. Also, update the title of the article. 

Thank you,

Lalith

**Comment by lalith1403** [comment]
The suggested topic is exciting. Look forward to reading it. Ping me once you update the topic proposal. We can then proceed with this topic.

**Comment by aidelojep** [comment]
I'm glad you find it exciting too. I'm charged up to jump on it already too. I have updated the topic proposal.

**Comment by lalith1403** [comment]
@aidelojep 
Sounds like a helpful topic - let's please be sure it adds value beyond what is in the official docs and what is covered in other blog sites.
Please be sure to double-check that it does not overlap with any existing EngEd articles or incoming topic suggestions (if you haven't already) to avoid any potential article closure, please reference any relevant EngEd articles in yours. 

Additionally, I would suggest paying keen attention to the grammar and formatting (there are a few formatting errors in the proposal) of the article to avoid any potential article closure.- Topic Approved :)

**Comment by aidelojep** [comment]
@lalith1403 noted with thanks. :)

**Comment by hectorkambow** [comment]
@aidelojep 
Just pinging you on this topic form - as I'm making some room in the topic queue.
No rush if you're still working on this one - if you need more time we can always re-open at a later date.

**Comment by hectorkambow** [comment]
@aidelojep 
Just pinging you on this topic form - as I'm making some room in the topic queue.
No rush if you're still working on this one - if you need more time we can always re-open at a later date

**Comment by aidelojep** [comment]
Hello Section-Engineering-Education/Engineering-Education,
I forked the repository and I am currently working on the project. Why was
it closed again?

I am set to complete this project before the weekend.

Best

Peter

**Comment by lalith1403** [comment]
You may re-open the issue before submitting the PR.

**Comment by aidelojep** [comment]
ok ,

Many thanks

**Comment by hectorkambow** [comment]
Just pinging you on this topic form - as I'm making some room in the topic queue.
No rush if you're still working on this one - if you need more time we can always re-open at a later date
